Makefile

```diff
@@ -44,12 +45,31 @@ $(call add-bindata,assets,./bindata/...,bindata,assets,pkg/operator/assets/binda
 $(call build-image,cert-manager-operator,$(IMAGE_OPERATOR),./images/ci/Dockerfile,.)
 $(call build-image,cert-manager-operator-bundle,$(IMAGE_OPERATOR_BUNDLE),./bundle/bundle.Dockerfile,./bundle)
 
+# generate scorecard image targets
+.PHONY: build/custom-scorecard-tests
+build/custom-scorecard-tests:
+	go build -mod=vendor $(GO_GCFLAGS) $(GO_ASMFLAGS) -o $(BUILD_DIR)/$(@F) ./test
+
+.PHONY: build-e2e-image
+build-e2e-image:
+	docker build -f ./images/custom-scorecard-tests/Dockerfile ./ -t quay.io/ytripath/cert-manager-operator-e2e-tests:latest
+	docker push quay.io/ytripath/cert-manager-operator-e2e-tests:latest
+
+.PHONY: run-e2e-image
+run-e2e-image: test-e2e-wait-for-stable-state
+	oc apply -f test/rbac.yaml
+	operator-sdk scorecard ./bundle --namespace=cm-tests --wait-time=600s -o json
+	oc delete -f test/rbac.yaml
+
 # exclude e2e test from unit tests
 GO_TEST_PACKAGES :=./pkg/... ./cmd/...
 
 # re-use test-unit target for e2e tests
-test-e2e: test-e2e-wait-for-stable-state
-	$(MAKE) GO_TEST_PACKAGES=./test/e2e/... test-unit
+test-e2e:
+	go build -o ./e2e-tests test/main.go
+	./e2e-tests TestSelfSignedCerts ./bundle
+	./e2e-tests TestACMECertsIngress ./bundle
+	./e2e-tests TestCertRenewal ./bundle
```

Comment:
Same as https://github.com/openshift/cert-manager-operator/pull/64/files#r871078707 , if 20 cases are added, current approach will make it have to enumerate 20 case names.

Reply:
I intend to remove this target, this only tests the binary, `make run-e2e-tests` actually runs the docker image